apps/builder/app/shared/instance-utils.test.ts

```diff
@@ -317,3 +329,91 @@ describe("find closest droppable target", () => {
     });
   });
 });
+
+test("insert template data with only new style sources", () => {
+  instancesStore.set(new Map([createInstancePair("body", "Body", [])]));
+  styleSourceSelectionsStore.set(new Map());
+  styleSourcesStore.set(
+    new Map([["1", { type: "token", id: "1", name: "Zero" }]])
+  );
+  stylesStore.set(
+    new Map([
+      [
+        "1:base:color:",
+        {
+          breakpointId: "base",
+          styleSourceId: "1",
+          property: "color",
+          value: { type: "keyword", value: "red" },
+        },
+      ],
+    ])
+  );
+  insertTemplateData(
+    {
+      children: [{ type: "id", value: "box1" }],
+      instances: [
+        { type: "instance", id: "box1", component: "Box", children: [] },
+      ],
+      props: [],
+      dataSources: [],
+      styleSourceSelections: [{ instanceId: "box1", values: ["1", "2"] }],
```

Comment:
"styleSourceSelection" seems more correct, because its one selection that contains many items

Reply:
It's an array of selections